src/configuration/evars.md

```diff
@@ -4,7 +4,7 @@ Below is a list of the most frequently used configuration variables. You can get
 
 The `e??` command to get help on all the evaluable configuration variables of radare2. As long as the output of this command is pretty large you can combine it with the internal grep `~` to filter for what you are looking for:
 
-![e??~color](../img/configuration/e--color.png)
+![e??~color](../img-white/configuration/e--color.png)
```

Comment:
Uhm. Img is a symlink to img-white. I thi k it will be better to remove the symlink and rename the folder properly. This was done for printing purposes to have all background of the inages in white